### PR TITLE
Fix a signed calculation error during loading screens

### PR DESCRIPTION
**Pull request by CakeTheLiar**
Inside the code for loading behavior files, Bethesda decided to read an unsigned short as a signed short and then sign extend it, which is then used in a calculation that crashes the game with too many animations (behaviors) installed.
This fix should simply allow for twice as many animations to be loaded.

For reference: from what I've gathered that variable must be hkbBehaviorGraph->numStaticNodes over in CommonLibSSE.

### CODE REVIEW

**Comment by aers** [comment]
I'll take a look in a few days when I have some free time, just commenting so you know I'm not ignoring you. o/

File: include/config.h [hunk 1]
```diff
@@ -36,6 +36,7 @@ namespace config
     extern bool fixSlowTimeCameraMovement;
     extern bool fixTreeReflections;
     extern bool fixVerticalLookSensitivity;
+	extern bool fixAnimationLoadSignedCrash;
```
Comment (AnyOldName3) [inline include/config.h]: Indentation mismatch.

File: src/config.cpp [hunk 2]
```diff
@@ -38,6 +38,7 @@ namespace config
     bool fixSlowTimeCameraMovement = true;
     bool fixTreeReflections = true;
     bool fixVerticalLookSensitivity = true;
+	bool fixAnimationLoadSignedCrash = true;
```
Comment (AnyOldName3) [inline src/config.cpp]: Indentation mismatch

File: src/fixes.cpp [hunk 3]
```diff
@@ -50,6 +50,9 @@ namespace fixes
         if (config::fixVerticalLookSensitivity)
             PatchVerticalLookSensitivity();
 
+		if (config::fixAnimationLoadSignedCrash)
+			PatchAnimationLoadSignedCrash();
```
Comment (AnyOldName3) [inline src/fixes.cpp]: Hey, guess what? Another indentation mismatch.

I guess at this point, you're going to know to look for more indentation mismatches on your own.

**Comment by CakeTheLiar** [comment]
Whoops, guess my VS was still set to use tabs instead of spaces. My bad.

**Comment by AnyOldName3** [comment]
It's aers' fault for not using an `.editorconfig` or `.clang-format` file to force your VS to switch.

**Comment by aers** [comment]
The game's updating in a few days so I'm not releasing an update until then, but this will obviously be in it.

**Pull request merged**